dashboard/app/models/pd/application/facilitator1819_application.rb

```diff
@@ -509,15 +525,32 @@ def to_csv_row(user)
 
     # @override
     def to_cohort_csv_row
+      columns = [
+        date_accepted,
+        applicant_name,
+        district_name,
+        school_name,
+        user.email,
+        status,
+        fit_workshop_date_and_location
+      ]
+      if optional_columns[:registered_workshop]
```

Comment:
Where does this `optional_columns` come from? Does it need to be passed in as a param? Add a test for this too

Reply:
oops, yes! thanks for noticing!